org.secretchronicles.TSC.json

```diff
@@ -0,0 +1,238 @@
+{
+    "app-id": "org.secretchronicles.TSC",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tsc",
+    "rename-icon": "tsc",
+    "rename-appdata-file": "tsc.appdata.xml",
+    "rename-desktop-file": "tsc.desktop",
+    "desktop-file-name-prefix": "(Nightly) ",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=x11",
+        "--socket=wayland"
+    ],
+    "cleanup": [
+        "/bin/mm-common-prepare",
+        "/bin/mrbc",
+        "/include/",
+        "/lib/cmake/",
+        "/lib/giomm-2.4/",
+        "/lib/glibmm-2.4/",
+        "/lib/pkgconfig/",
+        "/lib/sigc++-2.0/",
+        "/lib64/cmake/",
+        "/lib64/pkgconfig/",
+        "/share/aclocal/",
+        "/share/doc/",
+        "/share/man/",
+        "/share/mm-common/",
+        "/share/pkgconfig/",
+        "/share/SFML/",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --prefix=/app",
+                "./b2 install --with-atomic --with-chrono --with-date_time --with-filesystem --with-thread"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2",
```

Comment:
Is this the official Boost URL? If so, leave it so, if not: why not use the canonical project URL?

Reply:
Yeah, it's the official one. As in, if you go to the Boost website and right-click one of the downloads, this is what you get. They've moved from SourceForge to bintray for binary hosting.